openquake/server/fixtures/0001_cookie_consent.json

```diff
@@ -0,0 +1,64 @@
+[
```

Comment:
Should be better if this file will be moved below oq-platfrom-standalone repository because is there that is managed.